core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -734,6 +734,10 @@ private[kafka] class Acceptor(val endPoint: EndPoint,
         val endThrottleTimeMs = e.startThrottleTimeMs + e.throttleTimeMs
         throttledSockets += DelayedCloseSocket(socketChannel, endThrottleTimeMs)
         None
+      case e: IOException =>
+        info(s"Encounter IOException", e)
```

Comment:
nit: `Encountered IOException. Closing connection.`

Reply:
@showuon Thanks for the comment. I've updated the info message with the new commit